mods/ra/maps/allies-06a/allies06a-AI.lua

```diff
@@ -106,7 +106,15 @@ ProduceVehicles = function()
 	end)
 end
 
+shouldProduce = false
 ProduceNaval = function()
+	if #Utils.Where(Map.ActorsInWorld, function(self) return self.Owner == player and self.Type == "syrd" end) < 1 and shouldProduce == false then
```

Comment:
Don't take it wrong pls, but wouldn't it be better for performance to first check for shouldProduce = false and then the rest as an indented if statement? Because that binary check should be simpler than looking for shipyards every time. But I don't know if two Ifs don't do something to performance too, I am just asking :)

This is what I mean:
if shouldProduce == false then
    if #Utils.Where(Map.ActorsInWorld, function(self) return self.Owner == player and self.Type == "syrd" end) < 1 then
        Trigger.AfterDelay(DateTime.Minutes(1), ProduceNaval)
        return
    end
end

I hope I didn't make any mistake I am not 100% sure in lua.